package.json

```diff
@@ -78,6 +78,7 @@
     "@types/jest": "27.4.1",
     "@types/react": "17.0.43",
     "@types/react-dom": "17.0.14",
+    "@types/lodash.debounce": "4.0.6",
```

Comment:
There is no `package-lock.json` in this PR, I think there should be since you added things to the package.json